db/queries/user.sql

```diff
@@ -76,4 +76,19 @@ VALUES ($1, $2) ON CONFLICT DO NOTHING;
 
 -- name: AddUserToBranch :exec
 INSERT INTO branch_users (user_id, branch_id)
-VALUES ($1, $2) ON CONFLICT DO NOTHING;
\ No newline at end of file
+VALUES ($1, $2) ON CONFLICT DO NOTHING;
```

Comment:
`SELECT u.id, u.phone, u.first_name, u.last_name, u.created_at
FROM users u
LEFT JOIN user_roles ur ON u.id = ur.user_id
WHERE ur.user_id IS NULL;
`